ANMN/LTSP/TSaggregator/copyDataset.py

```diff
@@ -462,27 +477,41 @@
         elif var_order == 0:
             nc_variable_out[:] = ma_variable_all
 
-            # create the output global attributes
+            # update the output global attributes
             if hasattr(nc_variable_out, 'standard_name'):
                 if nc_variable_out.standard_name == 'latitude':
                     la_max = ma_variable_all.max(0)
                     la_min = ma_variable_all.max(0)
-                    nc_out.setncattr("geospatial_lat_max", la_max)
-                    nc_out.setncattr("geospatial_lat_min", la_min)
+                    gattr_tmp.update({"geospatial_lat_max": la_max})
+                    gattr_tmp.update({"geospatial_lat_min": la_min})
+                    #nc_out.setncattr("geospatial_lat_max", la_max)
+                    #nc_out.setncattr("geospatial_lat_min", la_min)
                 if nc_variable_out.standard_name == 'longitude':
                     lo_max = ma_variable_all.max(0)
                     lo_min = ma_variable_all.max(0)
-                    nc_out.setncattr("geospatial_lon_max", lo_max)
-                    nc_out.setncattr("geospatial_lon_min", lo_min)
+                    gattr_tmp.update({"geospatial_lon_max": lo_max})
+                    gattr_tmp.update({"geospatial_lon_min": lo_min})
+                    #nc_out.setncattr("geospatial_lon_max", lo_max)
+                    #nc_out.setncattr("geospatial_lon_min", lo_min)
                 if nc_variable_out.standard_name == 'depth':
                     d_max = ma_variable_all.max(0)
-                    d_min = ma_variable_all.max(0)
-                    nc_out.setncattr("geospatial_vertical_max", d_max)
-                    nc_out.setncattr("geospatial_vertical_min", d_min)
+                    d_min = ma_variable_all.min(0)
+                    gattr_tmp.update({"geospatial_vertical_max": d_max})
+                    gattr_tmp.update({"geospatial_vertical_min": d_min})
+                    #nc_out.setncattr("geospatial_vertical_max", d_max)
+                    #nc_out.setncattr("geospatial_vertical_min", d_min)
 
 
-nc.close()
+# sort new global attr dictionary
+gattr_new={}
+for key, value in sorted(gattr_tmp.items()):
+    gattr_new.update({key: value})
```

Comment:
This doesn't achieve the desired effect in Python v<3.6 because dict items are not ordered. But you can do away with the intermediate dict and add them to the file directly:
```python
for key, value in sorted(gattr_tmp.items()):
    nc_out.setncattr(key, value)
```